source/extensions/transport_sockets/tls/ssl_socket.h

```diff
@@ -86,9 +86,6 @@ class SslSocket : public Network::TransportSocket,
   void drainErrorQueue();
   void shutdownSsl();
   void shutdownBasic();
-  bool isThreadSafe() const {
-    return callbacks_ != nullptr && callbacks_->connection().dispatcher().isThreadSafe();
-  }
```

Comment:
Why we removed this method?

Reply:
It is only called in one place in ASSERT() which is not ignored in coverage tests. In order to raise the coverage, I inlined it instead.